clients.yml

```diff
@@ -1126,6 +1126,39 @@ clients:
   - 192.34.27.115
   - 2602:ffb1:0200:0:192:34:27:115
 
+- asn: "48717"
```

Comment:
What is this ASN? It hasn't been active since 2015 - https://bgp.he.net/AS48717

Reply:
AS48717 is my friend's ASN, he said it was in the routing table last week and was announcing something. I can remove it until he resolves it if you'd like

Reply:
yeah lets remove since it doesnt look active in any way

Reply:
Updated, included PR #5 as well